### DISCUSSION TITLE
Code of Conduct for Zigpy and CO

### DISCUSSION THREAD

**Discussion by MattWestb** [zigpy/zigpy#1145] [RFC]
Then Zigpy and all its libs is mostly used by (Z)HA i think it can being good using the same **Code of Conduct** for mot getting "broadcast storms" in the "mesh network".
https://github.com/home-assistant/core/blob/dev/CODE_OF_CONDUCT.md

Normally its no problems with user and devs is posting the the GIT with some exceptions / incidents. I like helping then i have the knowledge and most (around 95%) of all user is very thankfully then they is getting help and i like it.

Also i like its free saying what you think and being respected and most importing respecting other with not the same thinking and is normally no problems.

Then the world is continuing spinning and changing its being more problems with political things and some user (/devs) is not  like other political actions that have happening the last year.

I was trying moderating one situation that was happening but i dont like hurting nice persons and some was maintainers was moderating some post away.

For doing it easier then its next time is happening (its do sooner or later) its being better if Zigpy and CO is using the same **Code of Conduct** as HA and its only referring to it and dont need going in discussion with user / devs.

Thanks in advance for the great project and all the contribution that is being made for our systems and i like it continuing like that !!

MW

**Comment by dmulcahey** [comment]
Did we miss a bad situation somewhere?

**Reply by MattWestb** [reply]
It was one little not so nice one but not so many bad words but very "political East / West Europe" thing (no name here then im not taking part of it).

Some was flagging the key posting **destructive** (i cannot and i was flagging my post as of topic for not escalating it).
For my is **not** interesting if political, race, gender, and so on as long persons is nice.
And some time its no maintainer online or is digging in other things.

I only thinking its easier saying "pleas behave" and referring to **Code of Conduct** and its not escalating and destroying the "climate" for the users and devs (its one tool for solving "relation" problems).

I is not  being  hysterical with thins like "**black** / white  list" and so on and i leaving that to other to mange but i think Code of Conduct is one good thing and can helping then moderating the issues and discussions here.


PM I think im the only one that is permanent blacklisted (blocked) in deCONZ GIT for saying with i think and im not  sorry for that then persons and company is bad behaving.
